shared/modules/Numeric.ts

```diff
@@ -5,6 +5,8 @@ import { addHexPrefix } from 'ethereumjs-util';
 import { EtherDenomination } from '../constants/common';
 import { stripHexPrefix } from './hexstring-utils';
 
+const MAX_DECIMALS_FOR_TOKENS = 32;
+BigNumber.config({ DECIMAL_PLACES: MAX_DECIMALS_FOR_TOKENS });
```

Comment:
I don't see any upper limit on ERC-20 decimals, so it could be the case that 32 is still not enough for all tokens.  It would be good to run a script and see how many decimals contracts are using in practice.

The fix seems reasonable, but I'm not sure what else could break as a result.  (less performant due to more calculation? UI showing more decimals?  changes required in core/mobile?)

What's the root cause of not being able to send these tokens?  Like what error occurs when you try?  I'm wondering if we can still allow them to be sent, while keeping the lower 20 decimal precision.

Reply:
Also see this validation from core that requires decimals be from [0, 36]:

https://github.com/MetaMask/core/blob/014f6325e77afdc357ba1544a93c296dee9c0c56/packages/assets-controllers/src/assetsUtil.ts#L128-L133

Reply:
`What's the root cause of not being able to send these tokens? Like what error occurs when you try? I'm wondering if we can still allow them to be sent, while keeping the default 20 decimal precision.`
if i understand well , the suggestion is to see if it's cannot be fixed with another way ( from the UX for example